### PR TITLE
Improve and test the notify trigger

### PR DESCRIPTION
**Pull request by dgageot**
+ Mitigate the case where we miss the first change event
 + Add unit test

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -73,7 +73,8 @@
 
 [[constraint]]
   name = "github.com/rjeczalik/notify"
-  version = "0.9.2"
+  revision = "9ace999fc73bca667fd45dafc41356186f8b7e65"
+  source = "https://github.com/dgageot/notify.git"
```
Comment (nkubala) [inline Gopkg.toml]: can you add a comment here explaining what you fixed and why we're using your fork?

also, did you send a PR? :)

Comment (dgageot) [reply Gopkg.toml]: I’ll do that! Thanks!

Comment (dgageot) [reply Gopkg.toml]: In fact, with the mitigation code, I might get rid of the fork!

Comment (dgageot) [reply Gopkg.toml]: I got rid of the fork

File: pkg/skaffold/watch/triggers.go [hunk 2]
```diff
@@ -175,8 +180,11 @@ func (t *fsNotifyTrigger) Start(ctx context.Context) (<-chan bool, error) {
 				timer.Reset(t.Interval)
 			case <-timer.C:
 				trigger <- true
+			case <-initialTimer.C:
+				trigger <- true
```
Comment (balopat) [inline pkg/skaffold/watch/triggers.go]: shouldn't we have an initialTimer.Stop() here as well to trigger it only once?

**Pull request merged**